cmd/serverutil/serverutil.go

```diff
@@ -17,6 +17,7 @@ package serverutil
 
```

Comment:
I'm curious about how you're using the `cmd` pattern. Seems like this particular use is actually a library, rather than a binary?

Reply:
True, this breaks the pattern a bit, I could move to an internal directory?

Reply:
Yeah, I'd prefer if the `cmd` pattern be respected. `internal` would be nicer indeed - you don't need these to be publicly available, do you?